app/model_store.cpp

```
@@ -0,0 +1,1171 @@
+    wide.reserve(arguments.size() + 1);
+    wide.emplace_back(executable.wstring());
+    for (const auto& argument : arguments) wide.emplace_back(widen_utf8(argument));
+    std::vector<const wchar_t*> argv;
+    argv.reserve(wide.size() + 1);
+    for (const auto& argument : wide) argv.push_back(argument.c_str());
+    argv.push_back(nullptr);
+    const intptr_t status = _wspawnv(_P_WAIT, executable.c_str(), argv.data());
+    return status < 0 ? 127 : static_cast<int>(status);
+#else
+    const pid_t child = fork();
+    if (child < 0)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)